### PR TITLE
Updated readme 

### PR DESCRIPTION
**Pull request by HagenSR**
Updated readme with previous competitions for easy future reference

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,20 @@
...
+
+python .\launcher.pyz r - will run the game
+
+## Previous Byte-le Competitions
+
+2018 - Dungeon Delvers - https://github.com/jghibiki/Byte-le-Royale-2018
+
+2019 - Space Denizen - https://github.com/topoftheyear/Byte-le-Royale-2019
+
+2020 - Disaster Dispatcher - https://github.com/topoftheyear/byte_le_royale_2020
```
Comment (topoftheyear) [inline README.md]: 2020 is incorrect:
https://github.com/PixPanz/byte_le_royale_2020

**Review by topoftheyear — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (HagenSR) [reply README.md]: Fixed

**Review by topoftheyear — APPROVED** [review]
Very good. This is a smart idea, I'll get a link in here to the examples the instruction manual references in future PR.

**Comment by topoftheyear** [comment]
This PR is good to go for whoever has write access to it. It's just a readme change.

**Pull request merged**